app/controllers/katello/api/v2/distributions_controller.rb

```diff
@@ -1,38 +0,0 @@
-module Katello
-  class Api::V2::DistributionsController < Api::V2::ApiController
```

Comment:
Would it be a better idea to leave it in so the API functions as it did before? and that would mean leaving in the Distribution model and Glue::ElasticSearch::Distribution as well, right?